### ISSUE TITLE
Avoid loading Database on acts_as_tree declaration

### ISSUE THREAD

**Issue by matthewrudy** [amerine/acts_as_tree#54]
Right now when doing `rake assets:precompile` with no database
I get an exception:

```
PG::ConnectionBad: could not connect to server: No such file or directory
       	Is the server running locally and accepting
       	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
...
/usr/local/bundle/gems/activerecord-5.0.0.1/lib/active_record/attribute_methods/primary_key.rb:73:in `primary_key'
/usr/local/bundle/gems/acts_as_tree-2.5.0/lib/acts_as_tree.rb:68:in `acts_as_tree'
```

This was introduced in https://github.com/amerine/acts_as_tree/pull/52

And is caused by:

``` ruby
def acts_as_tree(options = {})
  configuration = {
    primary_key:   primary_key,
    ...
  }

  configuration.update(options) if options.is_a?(Hash)
  ...
end
```

Because that `primary_key` is always hit
I can't even override it.

Maybe

``` ruby
configuration = {}
configuration[:primary_key] = options[:primary_key] || primary_key
...
```

**Comment by felixbuenemann** [comment]
I think we can just set the default value to `"id"` and allow to override it in the options hash. 99% of the time the pk will be named "id" anyways.

**Comment by felixbuenemann** [comment]
@matthewrudy Fixed in v2.5.1. I just pushed the new gem to rubygems.

Thanks for bringing this to my attention!

**Comment by matthewrudy** [comment]
@felixbuenemann great
thanks for your rapid response!
